### ISSUE TITLE
bioconda/label/cf201901

### ISSUE THREAD

**Issue by nick-youngblut** [mmpust/raspir#7]
I'm just curious about the use of `bioconda/label/cf201901` in:

```
# Install your alignment tool of choice
# Burrows-Wheeler aligner [3]
conda install -c bioconda bwa
# Bowtie2 [4]
conda install -c bioconda/label/cf201901 bowtie2
```

Why that label? Are there particular versions of bwa or bowtie2 that will not work with raspir?

**Comment by nick-youngblut** [comment]
In other words, am I likely going to run into a problem with using the following to create the environment:

```
mamba create -n raspir bioconda::trimmomatic bioconda::samtools bioconda::bwa bioconda::bowtie2 conda-forge::pandas conda-forge::statsmodels conda-forge::matplotlib
```

**Comment by mmpust** [comment]
Dear Nick,
in the past, some bowtie2 users reported inter-deep conflicts during the initial bowtie2 test runs. These issues were solved when re-installing the software with the cf201901 label. However, for the alignment process, it may not be necessary to use the absolute latest bowtie2 code version, and therefore, I added the label in the description so that users may work with the package tree of January 2019 before conda switched to gcc7. Raspir should run either way. So, if you install the latest bowtie2 version and run into conflicts, then re-install bowtie2 with the cf201901 label and the issue should be solved.
Kind regards, 
Marie
